ner_v1/detectors/base_detector.py

```diff
@@ -13,21 +13,35 @@ class BaseDetector(object):
     detectors using translation
     
     Attributes:
-        supported_languages (set): Set of languages supported by a particular detector
-        source_language_script (str): ISO-639 language code of query text for detection
         target_language_script (str): ISO-639 language code in which detector would process the query
         translation_enabled (bool): Decides to either enable or disable translation API
     """
     __metaclass__ = abc.ABCMeta
 
-    supported_languages = set()
-    source_language_script = ENGLISH_LANG
     target_language_script = ENGLISH_LANG
     translation_enabled = False
```

Comment:
This point still stands, these seem like properties of individual detectors and should be instance attributes in my opinion instead of class level properties

Reply:
Also, these two can be protected level attributes.

Reply:
See comment below
https://github.com/hellohaptik/chatbot_ner/pull/74/files/5ac3098cc70fd8490976a11a2c808ec967dcc176..a473eba760d4654ecceed8c1400bf4133d66859d#r188511431